Formula/sfst.rb

```diff
@@ -0,0 +1,32 @@
+class Sfst < Formula
+  desc "Toolbox for morphological analysers and other FST-based tools"
+  homepage "http://www.cis.uni-muenchen.de/~schmid/tools/SFST/"
+  url "http://www.cis.uni-muenchen.de/~schmid/tools/SFST/data/SFST-1.4.7d.tar.gz"
+  sha256 "5a13c6a45298197216a6299eb6cdf96595d2036572bb518b9e1c1893cb1a6d5f"
+
+  def install
+    cd "src" do
+      system "make"
+      system "make", "DESTDIR=#{prefix}/", "install"
+      system "make", "DESTDIR=#{share}/", "maninstall"
+    end
+  end
+
+  test do
+    require "open3"
+    # this simple example is from the manual.
```

Comment:
You can probably remove these two comments, and leave an empty line between the `require` and the `(tespath` lines. I'm aware this is a fairly anal thing to care about 😅.